### PR TITLE
Sync Crowbar Devel and Staging directly from NUE (SOC-11396)

### PR DESCRIPTION
**Pull request by rtamalin**
Similarly to what was done in the past for Ardana, sync the Crowbar
Devel & Staging ISOs directly from Nuremberg source rather than from
the Provo mirrors.

Also use the same approach for the HOS 8 Devel mirroring.

### CODE REVIEW

**Review by jeremy-moffitt — APPROVED** [review]
Looks good to me, though I'd recommend waiting until Monday to merge it in case there are unexpected side effects

**Review by gosipyan — APPROVED** [review]

**Pull request merged**
